CMakeLists.txt

```diff
@@ -147,6 +147,7 @@ if (MFEM_USE_CUDA)
    find_package(CUDAToolkit REQUIRED)
    set(CUSPARSE_FOUND TRUE)
    get_target_property(CUSPARSE_LIBRARIES CUDA::cusparse LOCATION)
+   get_target_property(CUBLAS_LIBRARIES CUDA::cublas LOCATION)
```

Comment:
Similar to `CUSPARSE`, we need to have (when `find_package(CUDAToolkit REQUIRED)` succeeds)
```cmake
   set(CUBLAS_FOUND TRUE)
```
and also need to add `CUBLAS` to list(s) where `CUSPARSE` is already present.

Reply:
Done in bb8a5b88d.